### PR TITLE
Adding new event types to Alerts

### PR DESCRIPTION
**Pull request by gbstox**
Added Fields event & provider file events

### CODE REVIEW

File: docs/alerts_events.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ The types of events are grouped by the services that produce it.
...
 ### Uploaded File Processing Finished
 
-It happens when all steps of the processing of a machine operations file that was uploaded
+Event for when all steps of the processing of a machine operations file that was uploaded
 to Leaf are finished. This event can be useful to trigger a procedure that fetches the file from the API.
 This event can be used to activate a procedure that fetches the file from the api,
 since processing the file may take some time.
 
-The format of this alert event is:
+TAlerts for this event are in the following format:
```
Comment (Dsbaule) [inline docs/alerts_events.md:19]:
```suggestion
Alerts for this event are in the following format:
```

File: docs/alerts_events.md [hunk 2]
```diff
@@ -45,14 +45,70 @@ The format of this alert event is:
 }
 ```
 
+### Provider File Processing Finished
+
+Event for when the processing of an operations file from a provider 
+(either uploaded to provider mnaully or via Wireless Data Transfer) has successfully finished
```
Comment (Dsbaule) [inline docs/alerts_events.md:51]:
```suggestion
(either uploaded to provider manually or via Wireless Data Transfer) has successfully finished
```

File: docs/alerts_events.md [hunk 3]
```diff
@@ -45,14 +45,70 @@ The format of this alert event is:
...
+  "message": "details of what happened. May be empty",
+  "timestamp": "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'",
+  "type": "providerFileProcessingFinished"
+}
+```
+
+### Provider File Processing Failed
+
+Event for when the processing of an operations file from a provider 
+(either uploaded to provider mnaully or via Wireless Data Transfer) is finished
```
Comment (Dsbaule) [inline docs/alerts_events.md:69]:
```suggestion
(either uploaded to provider manually or via Wireless Data Transfer) is finished
```

**Review by Dsbaule — CHANGES_REQUESTED** [review]

File: docs/alerts_events.md [hunk 4]
```diff
@@ -45,14 +45,70 @@ The format of this alert event is:
...
+(either uploaded to provider mnaully or via Wireless Data Transfer) has successfully finished
+processing. This event can be useful for identifying new files when they are available.
+
+Alerts for this event are in the following format:
+
+```json
+{
+  "fileId": "the id of the uploaded file",
+  "leafUserId": "the id of the file owner",
+  "message": "details of what happened. May be empty",
```
Comment (doodomartins) [inline docs/alerts_events.md:60]: On success message we didn't return message.

```
{
  "leafUserId": "406faed2-366f-453b-a35c-3c7d1d0b58ce",
  "fileId": "b7a6741c-2bfe-4198-9e66-f4fe38104a2a",
  "type": "providerFileProcessingFinished",
  "timestamp": "2021-05-17T12:44:15.770603Z"
}
```

**Review by doodomartins — CHANGES_REQUESTED** [review]
Change the response on providerFileProcessingFinished

**Review by gbstox — COMMENTED** [review]
Accepted & fixed. Thank you!

**Review by Dsbaule — APPROVED** [review]

**Review by gbstox — COMMENTED** [review]
Updates alert response

**Review by doodomartins — APPROVED** [review]

**Pull request merged**